Templates/Multiplayer/Template/.gitattributes

```diff
@@ -0,0 +1,119 @@
+
+#
+# Git LFS (see https://git-lfs.github.com/)
+#
+*.3ds filter=lfs diff=lfs merge=lfs -text
```

Comment:
Should this fine be getting added?

Reply:
I think that's fine. Telling git that 3ds Autodesk files are not text files and should be controlled by LFS. Were you thinking of adding 3DS files to the ignore list instead since they are artist source assets, not a game engine asset?

Reply:
I meant file.

Reply:
Oh interesting, but dont we also want the gitattributes? It's useful if pull this repo down to work on the template itself, and it's also useful if I use this to create a new game if my game is using lfs?